src/controllers/article_tag.js

```diff
@@ -0,0 +1,26 @@
+import { Article } from '../db_api/models/index';
+
+const articleTag = async (req, res, next) => {
```

Comment:
@toluola Nice work done, but you can restructure your code better, make use of destructuring and other modern conventions to improve your code quality.

Reply:
fixed